### PR TITLE
CI: build armv8 64bit images

### PR DESCRIPTION
**Pull request by Williangalvani**
just creating a draft to check if CI finds some errors for us

### CODE REVIEW

**Comment by Williangalvani** [comment]
it doesnt show a manually built version on version chooser for me, which probably means the [architecture filter](https://github.com/Williangalvani/blueos-docker/blob/master/core/services/versionchooser/utils/dockerhub.py#L104) in there is workind.

**Comment by Williangalvani** [comment]
the tag for this branch doesnt show here though: https://hub.docker.com/layers/williangalvani/blueos-core/v8-goes-brrrrrr/images/sha256-706ba408ecf8d24cfc12a9091713006eca1c9ea5284a5cec079a383965f28363?context=explore

I suspect the one that finishes last is overwriting the first one, in CI

File: .github/workflows/test-and-deploy.yml [hunk 1]
```diff
@@ -95,7 +95,7 @@ jobs:
         docker: [bootstrap, core]
         project: [companion]
         new_project: [blueos]
-        platforms: ["linux/arm/v7"]
+        platforms: ["linux/arm/v7", "linux/arm64/v8"]
```
Comment (Williangalvani) [inline .github/workflows/test-and-deploy.yml:98]:
```suggestion
        platforms: ["linux/arm/v7,linux/arm64/v8"]
```

We'll need to run both in the same action, apparently. I'll do some reading on docs.

**Comment by Williangalvani** [comment]
@patrickelectric thoughts?

The issue here is that we cant do  the  `--output` line for two different images at the same time.

I think we could maybe do something like a `.join()` somewhere. or maybe we can use --output for two lines, but I couldn't find how to do it.

**Comment by Williangalvani** [comment]
nevermind me, [it seems to be possible](https://docs.docker.com/engine/reference/commandline/buildx_build/#output), let me troubleshoot it a bit further

**Comment by patrickelectric** [comment]
You can use a matrix and the name of the architecture in the artifact

**Comment by patrickelectric** [comment]
> > architecture
> 
> using a matrix causes 2 independent pushes to docker hub, where the latest one overwrites the first one

ouch, we can build both together and each individually for the artifacts, it should use the cache from both builds and just create it for us.

**Comment by Williangalvani** [comment]
> > > architecture
> > 
> > 
> > using a matrix causes 2 independent pushes to docker hub, where the latest one overwrites the first one
> 
> ouch, we can build both together and each individually for the artifacts, it should use the cache from both builds and just create it for us.

That is what the current change does. apparently we could use the "tar" output, but that doesn't seem to be "docker image specification compliant"

**Review by patrickelectric — APPROVED** [review]
I did some attempts to parallelize it but not ended well, I did updates and now appears to be fine.

**Comment by patrickelectric** [comment]
Be free to merge.

**Pull request merged**